### PR TITLE
Feature/abbrev review

### PR DESCRIPTION
**Pull request by KShih**
1. Display the average rate for this location
2. Display the review in card style

image

### CODE REVIEW

**Comment by ab7289** [comment]
image

Can we make this a scrollable container (similar to the table in the search page) so that it has a fixed height and if there are like 50 reviews that page isn't huge?

Also could we move the reviews container to the right hand side of the the screen? This way the user could see the apartments and reviews at the same time without having to scroll as much.

image

File: location/templates/location.html [hunk 1]
```diff
@@ -85,37 +85,53 @@ <h3>
...
+                                <span class="fa fa-star"></span>
+                            {% endif %}
                         {% endfor %}
-                    </tbody>
-                </table>
+                        </li>
+                    </ul>
+                </div>
+                <div class="card-body">
+                    <h5 class="card-title pricing-card-title">
```
Comment (mok232) [inline location/templates/location.html]: 👍

File: location/tests.py [hunk 2]
```diff
@@ -27,6 +29,22 @@ def test_google_map_url(self):
...
+        )
+        r2 = Review.objects.create(
+            location=l1, user=s1, content="test_content2", time=datetime.now(), rating=5
+        )
+        avg = (r1.rating + r2.rating) / 2
+        self.assertEqual(l1.avg_rate(), avg)
+
+    def test_empty_review_avg_rate(self):
+        l1 = Location.objects.create(state="NY")
+        self.assertEqual(l1.avg_rate(), 0)
```
Comment (mok232) [inline location/tests.py]: good test coverage!

**Comment by ab7289** [comment]
would it be possible to hide this text if the person is not logged in/not a tenant? I don't think it's really necessary.

image

**Comment by KShih** [comment]
@ab7289 Yes, but I leave this line either for demo usage or a reminder for the user like a renter.

**Pull request merged**